docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -0,0 +1,116 @@
+defines which Connector to be used to execute with given template.
+
+For example, see a relation between [Kafka element template](https://github.com/camunda/connectors/blob/main/connectors/kafka/element-templates/kafka-inbound-connector.json#L39)
+and [Kafka Connector](https://github.com/camunda/connectors/blob/main/connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java#L20).
+
+In order to make hybrid Connector runtime working correctly, it is required to override Connector type.
+
+For the purpose of this guide, let's consider we would like to override HTTP REST Connector with type `io.camunda:http-json:1`.
+Refer to the [element template](https://github.com/camunda/connectors/blob/main/connectors/http/rest/element-templates/http-json-connector.json#L50) and its related [runtime](https://github.com/camunda/connectors/blob/main/connectors/http/rest/src/main/java/io/camunda/connector/http/rest/HttpJsonFunction.java#L43).
+
+## Start Connectors runtime in a hybrid mode
+
+### Prerequisite
+
+You need to have a running Camunda cluster, and a pair of `Client ID`/`Client Secret` with `Zeebe` and `Operate` scopes.
+[Learn more](../components/console/manage-clusters/manage-api-clients/) about how to obtain required credentials.
+
+### Option A: get Connectors runtime from Docker registry
+
+Use this option when you don't need to make any code modifications to either Connectors runtime, or specific Connector.
+
+Run the following script:
+
+```shell
+docker run --rm --name=HybridConnector \
+    -e ZEEBE_CLIENT_SECURITY_PLAINTEXT=false \
+    -e ZEEBE_CLIENT_CLOUD_CLUSTER-ID='<YOUR_CLUSTER_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-ID='<YOUR_CLIENT_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-SECRET='<YOUR_CLIENT_SECRET>' \
+    -e ZEEBE_CLIENT_CLOUD_REGION='<YOUR_CLUSTER_REGION>' \
+    -e CAMUNDA_OPERATE_CLIENT_URL='https://<region>.operate.camunda.io/<cluster-id>' \
+    -e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:2' \
+        camunda/connectors-bundle:<desired-version>
+```
+
+### Option B: build your own runtime
+
+Use this option if you need to make code modifications.
+
+- Make sure `docker` is installed.
+- Clone https://github.com/camunda/connectors.
+- Go to `<connectors-root>/bundle/default-bundle`.
+- Build Connectors image, e.g. `docker build -f Dockerfile -t camunda/connectors-bundle:<desired-version> .`
+
+Run the following script:
+
+```shell
+docker run --rm --name=HybridConnector \
+    -e ZEEBE_CLIENT_SECURITY_PLAINTEXT=false \
+    -e ZEEBE_CLIENT_CLOUD_CLUSTER-ID='<YOUR_CLUSTER_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-ID='<YOUR_CLIENT_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-SECRET='<YOUR_CLIENT_SECRET>' \
+    -e ZEEBE_CLIENT_CLOUD_REGION='<YOUR_CLUSTER_REGION>' \
+    -e CAMUNDA_OPERATE_CLIENT_URL='https://<region>.operate.camunda.io/<cluster-id>' \
+    -e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:2' \
+        camunda/connectors-bundle:<desired-version>
+```
```

Comment:
We dont need to repear this as the script is the same as above.

Reply:
Done. Reworded with reference.